gitman/shell.py

```diff
@@ -118,11 +118,50 @@ def pwd(_show=True):
     return cwd
 
 
-def ln(source, target):
-    dirpath = os.path.dirname(target)
-    if not os.path.isdir(dirpath):
-        mkdir(dirpath)
-    os.symlink(source, target)
+def ln(source, target, symbolic):
```

Comment:
I think `symbolic` should be a required flag:

```suggestion
def ln(source, target, *, symbolic:bool):
```